### PR TITLE
Thumbnail in store rebased

### PR DESCRIPTION
**Pull request by beatthat**
These should be the changes we worked on together, rebased onto careerfair-integration branch

### CODE REVIEW

File: client/src/api.ts [hunk 1]
```diff
@@ -831,7 +831,7 @@ export async function uploadThumbnail(
       "Content-Type": "multipart/form-data",
     },
   });
-  return getDataFromAxiosResponse(result, []);
+  return getDataFromAxiosResponse(result, ["thumbnail"]);
```
Comment (beatthat) [inline client/src/api.ts:834]: this probably needs to be `['data', 'thumbnail']` (because this request isn't actually graphql)?

**Review by beatthat — COMMENTED** [review]
looks good...am curious about that dataPath (question inline)

Otherwise if works locally, let's merge and deploy on monday

File: client/src/api.ts (hunk 1 above)
Comment (supersweet-dev) [reply client/src/api.ts:834]: I actually as that and couldn't figure why it wasn't working but changing it to just "thumbnail" fixed it somehow
the actual response has the shape
data.data.thumbnail

Comment (beatthat) [reply client/src/api.ts:834]: The json in the response is just 

```json
{ "data": { "thumbnail": "" } }
```

...the first `data` property you're referring to is a `AxiosResponse` object

**Review by kycarr — APPROVED** [review]

**Pull request merged**